### DISCUSSION TITLE
Issues with the com frequencies on my Loupedeck

### DISCUSSION THREAD

**Discussion by Packman68** [calibx/msfsdeck#261] [Q&A]
This plug-in seems to work pretty well with MSFS 24, but when I change frequencies, it doesn’t really work well. The numbers jump around and it takes a while to change to a new frequency. Any suggestions?

**Comment by calibx** [comment]
Hi, the problem comes from the new way I handle the communication (1 loop in the plugin, 1 in the dll).
It lacks synchronization, creating that problem.
I will fix that with the first beta (after fixing disconnect / connect)

**Comment by Packman68** [comment]
No worries.  I love the rest of it!





On Monday, December 16, 2024, 1:15 PM, calibx ***@***.***> wrote:




Hi, the problem comes from the new way I handle the communication (1 loop in the plugin, 1 in the dll).
It lacks synchronization, creating that problem.
I will fix that with the first beta (after fixing disconnect / connect)
